extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -32,16 +32,6 @@ private function getTitleMock( $type, $langCode, $artId, $artDbKey ) {
 		return $title;
 	}
 
-	private function mockMonetizationModule() {
-		$name = 'MonetizationModuleHelper';
-		$mock = $this->getMock( $name );
-		$mock->method( 'getMonetizationUnits' )->willReturn( [ 'below_category' => 'testing' ] );
-		$mock->method( 'getWikiVertical' )->willReturn( 'other' );
-		$mock->method( 'getCacheVersion' )->willReturn( 'v1' );
-		$mock->method( 'canShowModule' )->willReturn( true );
-		$this->mockClass( $name, $mock );
-	}
-
 	public function adContextDataProvider() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "adContextDataProvider" has 184 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS138)